pkgs/by-name/mo/move-mount-beneath/package.nix

```
@@ -2,18 +2,17 @@
   lib,
   stdenv,
   fetchFromGitHub,
-  fetchpatch,
 }:
 
 stdenv.mkDerivation {
   pname = "move-mount-beneath";
-  version = "unstable-2023-11-26";
+  version = "unstable-2025-09-24";
```

```suggestion
  version = "0-unstable-2025-09-24";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning